### PR TITLE
Bookstore: add reducers and actions

### PR DESCRIPTION
**Pull request by AnicetFantomas**
In this pull request we performed :

- Understand the concept of state management.
- Use store, actions, and reducers in React.

### CODE REVIEW

**Review by nicupop729 — CHANGES_REQUESTED** [review]
Hi @AnicetFantomas, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request



You have submitted a project, that is missing the README file.
Kindly, please build the README file for this project in order for this milestone to be reviewed. Here is the template that needs to be followed [README Template](https://github.com/microverseinc/readme-template).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/App.js [hunk 1]
```diff
@@ -1,4 +1,3 @@
-import './App.css';
 import { Routes, Route } from 'react-router-dom';
```
Comment (mateo951) [inline src/App.js:1]: You followed the ducks pattern perfectly 💪 Well done ✔

File: src/Redux/configuration.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { combineReducers } from 'redux';
```
Comment (mateo951) [inline src/Redux/configuration.js]: Please import `createStore` from `redux` and use it to create a store through the reducer you created in line 5 📝

You could try something like:

    const store = createStore(rootReducer);

Don't forget to export your store as well 🤓

    export default store;

File: src/Redux/Books/Books.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    case ADD:
+      return {
+        ...state,
+        books: [state.title, state.author, action.type],
+      };
+    case REMOVE:
+      return {
+        ...state,
+        books: state.filter((book) => book.title !== action.type),
+      };
```
Comment (mateo951) [inline src/Redux/Books/Books.js:15]: Creates and Exports **Action Creators**. (`addBook` and `removeBook` functions) 📝 Good job!

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @AnicetFantomas, 

📝👌 **Your project highlights** 🔥😎 
- General Requirements ✔
- Added reducers and actions 🚀

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Review by BrianSammit — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello @AnicetFantomas, 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**